node/rpc/component.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 // Server constructs a new RPC Server from the given Config.
-// TODO @renaynay @Wondertan: this component is meant to be removed on implementation
-//  of https://github.com/celestiaorg/celestia-node/pull/506.
+//
+//	TODO(@renaynay/@Wondertan): this component is meant to be removed on implementation
+//	 of https://github.com/celestiaorg/celestia-node/pull/506.
```

Comment:
here too

Reply:
Has to be indented like this to both:
- be able to be part of a godoc comment (as a codeblock)
- and additionally be recognized as a multiline todo comment